src/tests/engines_util/CMakeLists.txt

```diff
@@ -10,5 +10,6 @@ ie_faster_build(engines_test_util UNITY)
 
 target_link_libraries(engines_test_util PUBLIC openvino::runtime openvino::runtime::dev gtest gmock ngraph_test_util)
 target_include_directories(engines_test_util PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories(engines_test_util PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/..")
```

Comment:
Why do you need this additional include directory?

Reply:
I added this to do not change source files like src/frontends/tests/shared/src/op_fuzzy.cpp:
#include "engines_util/test_case.hpp"

I can change sources instead of adding this include path. So please help to decide what approach is preferable.